docs/descriptor/includes/image/osbs.rst

```diff
@@ -18,10 +18,14 @@ It contains two main keys:
 .. code-block:: yaml
```

Comment:
The section above, `It contains two main keys:....` I think is also incorrect as OSBS contains 5 keys not 2. Would it be possible to amend the above as well please?

Reply:
Sure, done. I've added inter-links for all five sections too, underneath.

Reply:
I'd be grateful if someone could verify the added links are OK (I tried rendering the page using pandoc but it did not process the links)

Reply:
Links look fine - the `make preview` in the docs subdirectory helps to render them :-)